.circleci/config-2_1.yml

```diff
@@ -28,6 +28,12 @@ j8_small_par_executor: &j8_small_par_executor
     #exec_resource_class: xlarge
   parallelism: 1
 
+j8_small_executor: &j8_small_executor
+  executor:
+    name: java8-executor
+    exec_resource_class: medium
```

Comment:
can we do `#exec_resource_class: xlarge` like the rest? No executor selects medium, they rely on default.

Reply:
These tests are a super-tiny thing. I would rather be explicit as I _know_ they don't need xlarge rather than relying on defaults.